README.md

```diff
@@ -181,9 +181,9 @@ build_flags = -DNEOPIXEL_RGB -DDATA_PIN=2 ${env.build_flags} -DSECOND_SEGMENT_ST
 ...
 ```
 Implementation example:
-- The diagram of the board for WS2812b including ESP32 and the SN74AHCT125N 74AHCT125 [level shifter](https://github.com/awawa-dev/HyperHDR/wiki/Level-Shifter).
+- The diagram of the board for WS2812b/SK6812 including ESP32 and the SN74AHCT125N 74AHCT125 [level shifter](https://github.com/awawa-dev/HyperHDR/wiki/Level-Shifter).
 
-![HyperSPI](https://user-images.githubusercontent.com/85223482/216823785-746b940c-73ab-4b2f-a243-91dbc9d22816.png)
+![HyperSPI](https://user-images.githubusercontent.com/85223482/222923979-f344349a-1f8b-4195-94ca-51721923359e.png)
```

Comment:
@pavel-ponomarev asked me to add a new guide image to my request.

Reply:
Thanks, I reinstalled windows this week, removed all programs (Github Desktop).
Because of one change, I did not re-download it.

Reply:
@pavel-ponomarev, It is not necessary to install external applications to edit the 'readme.md' file.
You can edit it directly from the browser :)

@awawa-dev, on the page of the general [new manual](https://www.hyperhdr.eu/2023/02/ultimate-guide-on-how-to-build-led.html#chapter6) in this picture there is one extra board (Rpi4), in the background.

![Screenshot_1](https://user-images.githubusercontent.com/39379875/222925864-21c91ab1-2c89-4b30-b6ea-a8c0e9556db7.png)

Reply:
> You can edit it directly from the browser :)

Got it, I'll know for the future!